feature-libs/cart/import-export/components/import-export/import-export.module.ts

```diff
@@ -0,0 +1,43 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import {
+  CmsConfig,
+  ConfigModule,
+  I18nModule,
+  UrlModule,
+} from '@spartacus/core';
+import { PageComponentModule } from '@spartacus/storefront';
+import { CartTypes } from '@spartacus/cart/import-export/core';
+import { ExportEntriesModule } from '../export-entries';
+import { ImportToCartModule } from '../import-to-cart';
+import { ImportExportComponent } from './import-export.component';
+
+@NgModule({
+  imports: [
+    PageComponentModule,
+    ConfigModule.withConfig(<CmsConfig>{
+      cmsComponents: {
+        ImportExportComponent: {
+          component: ImportExportComponent,
+          data: {
+            importButtonDisplayPages: ['savedCarts', 'savedCartsDetails'],
+            exportButtonDisplayPages: ['savedCartsDetails', 'cart'],
```

Comment:
Let's be consistent and call it all `...Routes` (e.g. `importButtonDisplayRoutes`), but not `...Pages` (e.g. `importButtonDisplayPages`)